tests/FakeItEasy.Tests/Core/SequentialCallContextTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿namespace FakeItEasy.Tests.Core
+{
+    using System.Collections.Generic;
+    using FakeItEasy.Core;
+    using FakeItEasy.Tests.TestHelpers;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class SequentialCallContextTests

```

Comment:
This class seems to be testing a number of things, rather than just the `SequentialCallContext`, so the name is slightly misleading.

More generally, is it offering anything over and above `OrderedCallMatchingSpecs`?